frontend/webpack.common.js

```diff
@@ -32,6 +32,10 @@ module.exports = {
         test: /\.(jpg|png|gif|svg)$/,
         type: 'asset/resource',
       },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/i,
+        type: 'asset/resource',
+      },
```

Comment:
이거 저번에 급하게 넣었긴한데, 합쳐도 될 것 같아요
```suggestion
        test: /\.(jpg|png|gif|svg|woff|woff2|eot|ttf|otf)$/,
        type: 'asset/resource',
      },

```